packages/react-core/src/components/Form/FormGroup.tsx

```diff
@@ -40,24 +45,34 @@ export const FormGroup: React.FunctionComponent<FormGroupProps> = ({
   hasNoPaddingTop = false,
   helperText,
   helperTextInvalid,
+  helperTextIcon,
+  helperTextInvalidIcon,
   fieldId,
   ...props
 }: FormGroupProps) => {
-  const validHelperText = (
-    <div
-      className={css(styles.formHelperText, validated === ValidatedOptions.success && styles.modifiers.success)}
-      id={`${fieldId}-helper`}
-      aria-live="polite"
-    >
-      {helperText}
-    </div>
-  );
+  const validHelperText =
+    helperText && (helperText as React.ReactElement).type === FormHelperText ? (
```

Comment:
Change to `React.isValidElement(helperText) && (helperText as React.ReactElement).type === FormHelperText`